modules/oauth-api/src/main/java/org/mule/runtime/oauth/api/PlatformManagedConnectionDescriptor.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.oauth.api;
+
+import org.mule.api.annotation.Experimental;
+import org.mule.api.annotation.NoExtend;
+
+import java.util.Map;
+
+/**
+ * Describes the configuration of an OAuth connection managed by the Anypoint Platform.
+ * <p>
+ * Platform Managed OAuth is an experimental feature. It will only be enabled on selected environments and scenarios.
+ * Backwards compatibility is not guaranteed.
+ *
+ * @since 4.3.0
+ */
+@NoExtend
+@Experimental
+public interface PlatformManagedConnectionDescriptor {
+
+  /**
+   * @return the connection's unique id in the Anypoint Platform
+   */
+  String getId();
+
+  /**
+   * @return the URI that identifies this connection in the Anypoint Platform
+   */
+  String getUri();
+
+  /**
+   * @return The name that the user has given this connection in the Anypoint Platform
+   */
+  String getName();
+
+  /**
+   * Returns parameters additional to the standard OAuth ones that the user has configured for this connection. These usually
+   * correspond to custom parameters tha the service provider accepts or requires on its requests, but may contain other
+   * parameters as well.
+   *
+   * @return A {@link Map} which keys are the parameter names
+   */
+  Map<String, Object> getParameters();
```

Comment:
parameters is to generic. Can this be renamed to `getCustomParameters` or `getUserParameters`?

Reply:
it's actually intentional. I didn't want it to be too specific as this might have unknown uses in the future. It's the same logic I asked you to follow yesterday on your PR's. If I change it as you ask, I not only risk having an API naming problem in the future, I also risk you asking me the same things I ask of you yesterday.